src/main/java/it/pagopa/transactions/commands/handlers/TransactionRequestAuthorizationHandler.java

```diff
@@ -65,12 +66,20 @@ public Mono<RequestAuthorizationResponseDto> handle(TransactionRequestAuthorizat
 
                     return monoPostePay.switchIfEmpty(monoXPay)
                             .flatMap(tuple2 -> {
+
                                 TransactionAuthorizationRequestedEvent authorizationEvent = new TransactionAuthorizationRequestedEvent(
                                         transaction.getTransactionId().value().toString(),
-                                        transaction.getRptId().value(),
-                                        transaction.getTransactionActivatedData().getPaymentToken(),
+                                        transaction.getNoticeCodes().stream().map(
+                                                noticeCode ->  new NoticeCode(
+                                                        transaction.getTransactionActivatedData().getNoticeCodes().
+                                                                stream().filter(noticeCode1 -> noticeCode1.getRptId().equals(noticeCode.rptId().value())).findFirst().get()
+                                                                .getPaymentToken(),
```

Comment:
same as in src/main/java/it/pagopa/transactions/commands/handlers/TransactionAddUserReceiptHandler.java